espanol/Tyrion/Daenerys/Daenerys.md

```diff
@@ -0,0 +1 @@
+LLegó Daenerys montada en su dragón, y con una espectacular entrada les sonrío. Unos cayeron enamorados iso-facto pero otros la odiarion desde el principio. Nadie quedó impasible.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.